.coveragerc

```diff
@@ -21,7 +21,7 @@ omit =
   google/cloud/__init__.py
 
 [report]
-fail_under = 99
```

Comment:
I think the the `fail-under` in `noxfile.py` also needs to be raised to 100.
https://github.com/googleapis/python-bigtable/blob/a2f89f4bfc8e9f240a26709da804b0324f321389/owlbot.py#L86-L90
 https://github.com/googleapis/python-bigtable/blob/a2f89f4bfc8e9f240a26709da804b0324f321389/noxfile.py#L201

Reply:
Ugh, seems like we should have a single source of truth for that. 5cb7ddf